lib/restforce/db/cleaner.rb

```diff
@@ -59,7 +59,7 @@ def valid_salesforce_ids
         valid_ids = []
 
         @runner.run(@mapping) do |run|
-          run.salesforce_records { |record| valid_ids << record.id }
+          run.salesforce_instances { |instance| valid_ids << instance.id }
```

Comment:
I agree wholehartedly; I have some stashed code on my machine to introduce request caching which already takes care of most of that. I'll have another PR out with those changes in the not-too-distant future.